STORM-IRIT/scripts/run.sh

```diff
@@ -8,18 +8,24 @@ process(){
     path=output/$filename
     
     python3 preprocess_cloud.py $base_name
-    ./pdpcComputeMultiScaleFeatures -i $path/"${filename}.ply" -v -o $path/$filename
-    ./pdpcSegmentation -i $path/"${filename}.ply" -s $path/"${filename}_scales.txt" -f $path/"${filename}_features.txt" -o $path/$filename -v
+    ./pdpcComputeMultiScaleFeatures -i "${filename}.ply" -v -o $path/$filename
```

Comment:
`-o $path/$filename` you still use output dir for tmp files

Reply:
fixed